entries/m/my.plesk.com.json

```
@@ -0,0 +1,11 @@
+{
+  "My Plesk": {
+    "domain": "my.plesk.com",
+    "tfa": [
+      "totp"
+    ],
+    "keywords": [
+      "hosting"
```

```suggestion
      "remote"
```
Given that My Plesk is neither a hosting provider or a VPS service I think Remote Access is a better category. Would you agree?